scripts/update_android_ios_dependencies.py

```diff
@@ -34,18 +34,30 @@
 Usage:
 # Dryrun (does not update any files on disk but prints out all replacements for
 # preview) - Update versions in default set of files in the repository.
-python3 scripts/update_ios_android_dependencies.py --dryrun
+python3 scripts/update_android_ios_dependencies.py --dryrun
 
-# Update versions in default set of files in the repository
-python3 scripts/update_ios_android_dependencies.py
+# Update versions in default set of files in the repository.
+python3 scripts/update_android_ios_dependencies.py
+
+# Update versions in default set of files in the repository. (allow updating to
+# experimental versions)
+python3 scripts/update_android_ios_dependencies.py --allow_experimental
 
 # Update only Android packages
-python3 scripts/update_ios_android_dependencies.py --skip_ios
+python3 scripts/update_android_ios_dependencies.py --skip_ios
 
 # Update specific pod files (or directories containing pod files)
-python3 scripts/update_ios_android_dependencies.py --podfiles foo/Podfile
+python3 scripts/update_android_ios_dependencies.py --podfiles foo/Podfile
                                                               dir_with_podfiles
 
+# Update all Android packages except any names containing androidx and auth
+python3 scripts/update_android_ios_dependencies.py --skip_ios
+  --ignore_android_packages auth androidx
+
+# Ignore updating any files in gameloop_android and test directories
+python3 scripts/update_android_ios_dependencies.py --ignore_directories test
```

Comment:
I feel it's safe to ignore the  /scripts folder

Reply:
Yup. Makes sense. I want to avoid hard coding as much as possible. This docstring is an example of the flags that users can use to ignore specific directories.